I got to a point where the build succeeds, but it fails to upload not due to missing credentials, but to invalid credentials:
```
2020-08-04T17:19:09.4304530Z Build 2.16.0 is already defined in versions endpoint.
2020-08-04T17:19:09.4304830Z The new build would be uploaded onto the same URL.
2020-08-04T17:19:09.4306290Z Uploading 2.16.0 build
...
2020-08-04T17:19:14.2060760Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/ja.lproj/GMSCore.strings
2020-08-04T17:19:14.2066770Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/el.lproj/GMSCore.strings
2020-08-04T17:19:14.2071250Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/ar.lproj/GMSCore.strings
2020-08-04T17:19:14.2075350Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/en_AU.lproj/GMSCore.strings
2020-08-04T17:19:14.2079130Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/he.lproj/GMSCore.strings
2020-08-04T17:19:14.2083430Z a ./GoogleMaps.bundle/GMSCoreResources.bundle/de.lproj/GMSCore.strings
2020-08-04T17:19:14.2100190Z a ./_CodeSignature/CodeResources
2020-08-04T17:19:14.3714650Z There was an error running client-build command: Access Denied
2020-08-04T17:19:14.3714840Z 
2020-08-04T17:19:14.3784060Z ##[error]Process completed with exit code 1.
```

If not AWS credentials from Github secrets vault then… which ones? Please, help, @nicknovitski! 🙏

--- a/.github/workflows/client-ios.yml
+++ b/.github/workflows/client-ios.yml
@@ -65,8 +65,8 @@ jobs:
       - run: expotools client-build --platform ios
         timeout-minutes: 60
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_ACCESS_KEY_ID: AKIAJ3SWUQ4QLNQC7FXA
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.android_client_build_aws_secret_key }}
       - name: Save test results
         if: always()
         uses: actions/upload-artifact@v2
@@ -76,8 +76,8 @@ jobs:
       - run: expotools client-build --platform ios --release # should only upload already-built app
         if: ${{ github.event.inputs.releaseSimulator == 'release-simulator' }}
         env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_ACCESS_KEY_ID: AKIAJ3SWUQ4QLNQC7FXA
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.android_client_build_aws_secret_key }}
       - uses: 8398a7/action-slack@v3
         if: failure() && (github.event.ref == 'refs/heads/master' || startsWith(github.event.ref, 'refs/heads/sdk-'))
         env:

